### DISCUSSION TITLE
PIDs for 2020 Ford F450s, etc.

### DISCUSSION THREAD

**Discussion by coogle** [meatpiHQ/wican-fw#133] [General]
Hey all,

So I got my WiCAN all set up and pulling some interesting data from my 2020 Ford F450 using the ODB-II standard list of PIDs. One thing I really wanted, and I am surprised I can't get, is my Odometer reading. Trying to figure out why it seems that maybe this is a custom PID that I don't know about?

Can anyone point me in the right direction here? I've been googling for hours looking for an answer but I'm not sure I'm asking the right questions. 

Any help appreciated!

**Comment by meatpiHQ** [comment]
@coogle Some cars do not support Odometer PID, my car is the same. (Kona 2021)

Can you read the Odometer from carscanner or some other app?

**Comment by coogle** [comment]
No, but I just assumed the problem there was that I didn't know the correct PID incantation to request the reading and that Ford decided not to use the ODB2 standard PID of 166 for it?

**Reply by meatpiHQ** [reply]
It possible that they are using a custom PID. In some cases app such as carscanner or others might know these custom PID. I've seen this so many times.

**Reply by meatpiHQ** [reply]
If you find it please post it here, other users might find it helpful.
